vocabularies/telephone-type.ttl

```diff
@@ -0,0 +1,41 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix tsl: <http://linked.data.gov.au/def/telephone-type/> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+<http://linked.data.gov.au/def/telephone-type> a owl:Ontology,
+        skos:ConceptScheme ;
+    dcterms:created "2020-03-25"^^xsd:date ;
+    dcterms:creator <http://linked.data.gov.au/org/gsq> ;
+    dcterms:modified "2020-03-25"^^xsd:date ;
+    dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+    dcterms:source "Derived from Australian Standard AS 4590.1:2017" ;
```

Comment:
If we are now referencing  VCARD our source should probably indicate this.

Reply:
No, I don't think so. `dcterms:source` should be used to indicate any content, RDF or otherwise, from which this vocab is delivered however in this case we aren't *deriving* from VCARD but rather doing the normal RDF thing of reusing VCARD elements, just as we reuse SKOS and OWL elements, so I think no source property is necessary.

We could use source if there was no other way to know where content came from but with RDF reuse, we have the URIs.